src/Packetery/Module/Checkout.php

```diff
@@ -987,8 +988,12 @@ public function getShippingRates( ?array $allowedCarrierNames ): array {
 			$cost = $this->getRateCost( $options, $cartPrice, $totalCartProductValue, $cartWeight );
 			if ( null !== $cost ) {
 				$carrierName = $this->getFormattedShippingMethodName( $carrierName, $cost );
-				$rateId      = ShippingMethod::PACKETERY_METHOD_ID . ':' . $optionId;
-				$taxes       = null;
+				if ( $this->options_provider->isWcCarrierConfigEnabled() ) {
+					$rateId = BaseShippingMethod::PACKETA_METHOD_PREFIX . $carrier->getId() . ':' . $optionId;
```

Comment:
raději bych viděl u spojování takhle stringu.. líp se mi to čte, ale to je asi spíš osobní preference.

Reply:
Po dohodě necháme takto.